lib/SizeLimitedHash.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace PostHog;
+
+class SizeLimitedHash
```

Comment:
are there tests for this to ensure the size never exceeds `SIZE_LIMIT` ?

Reply:
yup `testEventCalled`